x-pack/test_serverless/api_integration/test_suites/common/telemetry/telemetry_config.ts

```diff
@@ -5,14 +5,16 @@
  * 2.0.
  */
 
-import { expect } from 'expect';
+import { expect } from '@kbn/expect';
 import { SupertestWithRoleScopeType } from '@kbn/test-suites-xpack/api_integration/deployment_agnostic/services';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
 export default function telemetryConfigTest({ getService }: FtrProviderContext) {
   const roleScopedSupertest = getService('roleScopedSupertest');
   let supertestAdminWithApiKey: SupertestWithRoleScopeType;
   let supertestAdminWithCookieCredentials: SupertestWithRoleScopeType;
+  const retry = getService('retry');
+  const retryTimeout = 180 * 1000;
```

Comment:
IMO, even a shorter timeout would be enough (we poll every 10s). 20s should be plenty.

Reply:
https://github.com/elastic/kibana/pull/197036/commits/3a1d5d57e6d022bbcb6b76bd07bc50c1a3b75c91